decorators.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.chars.map(&:upcase).join
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0, 9)
```

Comment:
- [ ] Please ensure that you replace the `9` with `10` so that it will display `10` characters as oppose to `9` characters as part of the project requirement.